package.json

```diff
@@ -5,29 +5,34 @@
   "main": "dist/videojs-contrib-quality-levels.cjs.js",
   "jsnext:main": "src/plugin.js",
   "generator-videojs-plugin": {
-    "version": "7.3.2"
+    "version": "8.0.0"
   },
   "repository": "videojs/videojs-contrib-quality-levels",
   "scripts": {
     "prebuild": "npm run clean",
-    "build": "npm-run-all -p build:*",
+    "build": "npm-run-all -s clean -p build:*",
+    "build-prod": "cross-env-shell NO_TEST_BUNDLE=1 'npm run build'",
+    "build-test": "cross-env-shell TEST_BUNDLE_ONLY=1 'npm run build'",
     "build:js": "rollup -c scripts/rollup.config.js",
-    "clean": "shx rm -rf ./dist ./test/dist",
+    "clean": "shx rm -rf ./dist ./test/dist ./cjs ./es && shx mkdir -p ./dist ./test/dist ./cjs ./es",
     "postclean": "shx mkdir -p ./dist ./test/dist",
     "docs": "npm-run-all docs:*",
     "docs:api": "jsdoc src -c scripts/jsdoc.config.json -r -d docs/api",
-    "docs:toc": "doctoc README.md",
+    "docs:toc": "doctoc --notitle README.md",
     "lint": "vjsstandard",
     "server": "karma start scripts/karma.conf.js --singleRun=false --auto-watch",
     "start": "npm-run-all -p server watch",
     "pretest": "npm-run-all lint build",
-    "test": "karma start scripts/karma.conf.js",
+    "test": "npm-run-all lint build-test && karma start scripts/karma.conf.js",
+    "posttest": "shx cat test/dist/coverage/text.txt",
     "update-changelog": "conventional-changelog -p videojs -i CHANGELOG.md -s",
+    "preversion": "npm test",
     "version": "is-prerelease || npm run update-changelog && git add CHANGELOG.md",
     "watch": "npm-run-all -p watch:*",
+    "watch:cjs": "npm run build:cjs -- -w",
```

Comment:
I just noticed that this and the script below fail because there's no `build:cjs` or `build:es` script. Are these necessary or is `watch:js` sufficient?

Reply:
Thanks. This was just because I'd selected a wrong option when running the latest plugin generator, and manually reverted the changes that made. But missed this bit.